### ISSUE TITLE
We do not have loss operators in CPP?

### ISSUE THREAD

**Issue by zhangchen-qinyinghua** [dmlc/mxnet#897]
I can't find the loss operators in CPP codes, this makes it hard to control the backward process (in c/cpp).
Also, the trainning process seems to be sealed inside the python object mxnet.model.

All the above mean it is not easy to control the training process. For example, is it easy to give different learning rates to different layers?
so...I don't feel that a good practice.

Correct me if I missunderstood anything.

**Comment by zhangchen-qinyinghua** [comment]
Seems the softmax-output operator is just the loss operator.
I will close this issue.
